amp/WEB-INF/src/org/digijava/module/aim/util/AuditLoggerUtil.java

```diff
@@ -402,6 +403,26 @@ public static void deleteLogsByPeriod(String interval) {
         }
     }
 
+
+    public static List<Object[]> getListOfActivitiesFromAuditLogger2() {
+        String query = "select * from ( "
+                + " select  aav.amp_activity_id current_id ,  "
+                + " lead(aav.amp_activity_id, 1) "
```

Comment:
shouldn't this be LAG(aav.amp_activity_id, 1) instead?

Reply:
You are right. I did a last minute change and change the order by here `over(partition by aav.amp_activity_group_id order by aav.amp_activity_id asc)` having the order asc we should use lag. I had desc thats why iI used lead. 
We can either use lag with asc, or lead with desc.
Good Catch!